### PR TITLE
Qa fix/collection header public profile link

### PR DESCRIPTION
**Pull request by rej156**

### CODE REVIEW

File: packages/kauri-web/components/containers/Collection/View.js [hunk 1]
```diff
@@ -34,42 +34,44 @@ const HeaderContainer = styled(ContentContainer)`
...
-  render() {
+  render () {
     if (!this.props.data || !this.props.data.collection) return null
     const { name, background, description, date_created, date_updated, owner, sections } = this.props.data.collection
-    const { routeChangeAction } = this.props;
-    const myKeywords = rake(description, { language: 'english' });
-    const bg = background ? background : this.state.trianglifyBg;
+    const { routeChangeAction } = this.props
+    const myKeywords = rake(description, { language: 'english' })
+    const bg = background || this.state.trianglifyBg
```
Comment (davodesign84) [inline packages/kauri-web/components/containers/Collection/View.js]: nice

File: packages/kauri-components/components/Headers/CollectionHeader.re [hunk 2]
```diff
@@ -60,22 +49,49 @@ let make =
...
-        />
+        {
+          Belt.Option.mapWithDefault(
+            linkComponent,
+            <UserWidgetSmall
+              pageType=None
+              username=username->Belt.Option.getWithDefault(userId)
+              color="ffffff"
+            />,
+            linkComponent =>
```
Comment (davodesign84) [inline packages/kauri-components/components/Headers/CollectionHeader.re]: Just import the link component in here instead?

Comment (rej156) [reply packages/kauri-components/components/Headers/CollectionHeader.re]: Nah storybook is a bitch

Comment (rej156) [reply packages/kauri-components/components/Headers/CollectionHeader.re]: Hence the option

**Pull request merged**